src/storage/page_buffer.h

```diff
@@ -460,5 +460,6 @@ extern void pgbuf_daemons_destroy ();
 #endif /* SERVER_MODE */
 
 extern int pgbuf_start_scan (THREAD_ENTRY * thread_p, int type, DB_VALUE ** arg_values, int arg_cnt, void **ptr);
+extern void cast_pgptr_to_iopgptr (FILEIO_PAGE * &io_page, PAGE_PTR page_ptr);
```

Comment:
- use pbguf_ suffix for page buffer functions.
- Since it is pgptr to iopgptr, it is expected the that first argument is pgptr and second argument is io_page.
- Reference to pointer is C++ syntax and may confuse indent; disable it. Alternatively, use the REFPTR macro from porting.h.
```suggestion
// *INDENT-OFF*
extern void pgbuf_pgptr_to_iopgptr (PAGE_PTR page_ptr, FILEIO_PAGE *&io_page);
// *INDENT-ON*
```